.github/workflows/extensive_ci.yml

```diff
@@ -165,15 +165,19 @@ jobs:
 
     - name: Install simsopt package
       if: contains(matrix.packages, 'spec') || contains(matrix.packages, 'all')
-      run: pip install -v .[MPI,SPEC]
+      run: |
+        pip install -v .
+        pip install mpi4py py_spec pyoculus h5py
```

Comment:
What wasn't working about the original line here? Doesn't `[MPI,SPEC]` work for installing `mpi4py py_spec pyoculus h5py`?

Reply:
Yes. Not sure why.